### DISCUSSION TITLE
To keep an eye on: ad conversion and privacy sandbox

### DISCUSSION THREAD

**Discussion by uazo** [bromite/bromite#928] [Show and tell]
maybe someone knows something, I found this

https://www.w3.org/2019/09/18-ad-privacy-minutes.html
https://chromestatus.com/feature/6412002824028160
https://groups.google.com/a/chromium.org/g/blink-dev/c/7B0ldtZR_68/m/GjLBu0n4DgAJ
https://github.com/WICG/conversion-measurement-api

related code: 
https://chromium-review.googlesource.com/q/hashtag:privacy-sandbox
https://chromium-review.googlesource.com/c/chromium/src/+/1967220

to be studied, maybe someone can make a summary.

**Comment by csagan5** [comment]
See also:
* https://www.zdnet.com/article/google-sued-by-us-department-of-justice-in-search-monopoly-advertising-case/
* https://www.euronews.com/2020/12/17/google-sued-by-10-us-states-for-anti-competitive-online-ad-sales

I think this is related to that: Google plans to block third-party cookies altogether, and the move will hurt smaller players but not them because of this new "privacy sandbox" feature.

@uazo we can already disable it, there should be no breakage on websites because of it.

**Reply by csagan5** [reply]
Note: it is not yet there in v88

**Reply by uazo** [reply]
> we can already disable it,

I don't know if it is the right approach.

if, for example, the opt-in to this feature reduce the use of supercookies, we could enable it and deny the sending of measurement reports (or reduce the uid to 8 bit as suggested by apple and mozilla, if failure to send would recognize bromite as browser).
Or, via userscripts, we could inject some css/js to hide or disable all those tags with the attributes defined in https://github.com/WICG/conversion-measurement-api, considering that our adblocker only works on fetches, and not directly on html.

**Reply by csagan5** [reply]
> if failure to send would recognize bromite as browser

In this new proposal, are users allowed to opt out altogether? They must be (this is my point of view), and Bromite should have the attribution reporting disabled by default. It would not recognise Bromite any differently than it already happens because of the ad blocker, and the parties are the same.

Then we leave the possibility to re-enable the attribution to the user.

**Comment by samuraikid0** [comment]
What is google privacy sandbox? (Google and privacy two words that should never be on the same text)

https://mikestone.me/googles-privacy-sandbox/

**Comment by csagan5** [comment]
A post from EFF: https://www.eff.org/deeplinks/2021/03/googles-floc-terrible-idea

**Reply by ghost** [reply]
Eff only criticises chrome/chromium but bot firefox. This dual behaviour makes their opinion of less importance
